exercises/isbn-verifier/description.md

```diff
@@ -9,27 +10,30 @@ The program should allow for ISBN-10 without the separating dashes to be verifie
 
```

Comment:
While we're here: Line 8 is grammatically odd.
Also this is the first indication that there could be dashes in the number.
Maybe add an example number to the first paragraph.

Reply:
Done :+1: